arch/arm/boot/dts/overlays/rotary-encoder-overlay.dts

```diff
@@ -0,0 +1,27 @@
+// Device tree overlay for GPIO connected rotary encoder.
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
```

Comment:
I think rotary encoder done this for us. when I load the original PR.
```
# cat /sys/kernel/debug/pinctrl/3f200000.gpio/pinmux-pins

Pinmux settings per pin
Format: pin (name): mux_owner gpio_owner hog?
pin 0 (gpio0): (MUX UNCLAIMED) (GPIO UNCLAIMED)
pin 1 (gpio1): (MUX UNCLAIMED) (GPIO UNCLAIMED)
pin 2 (gpio2): (MUX UNCLAIMED) (GPIO UNCLAIMED)
pin 3 (gpio3): (MUX UNCLAIMED) (GPIO UNCLAIMED)
pin 4 (gpio4): (MUX UNCLAIMED) pinctrl-bcm2835:4
.
.
.
pin 17 (gpio17): (MUX UNCLAIMED) pinctrl-bcm2835:17
```
We can see pin 4 and pin 17 which used by rotary encoder already claimed by pinctrl-bcm2835:4.
And when I try to export pin 4.
```
# echo 4 >> /sys/class/gpio/export 
-bash: echo: write error: Device or resource busy
```